frontend/src/App.vue

```diff
@@ -3,6 +3,7 @@
   <v-app>
     <router-view-transition is-root />
     <snackbar />
-    <player-element />
+    <pip-video-player />
```

Comment:
I think this should live in the default layout. Doesn't make sense to have the pip player ready on the login and music playback layouts.

Reply:
Done @ferferga